src/components/menu/menu.tsx

```diff
@@ -84,6 +84,9 @@ export class Menu {
     }
 
     public render() {
+        const dropdownZIndex = getComputedStyle(this.host).getPropertyValue(
+            '--dropdown-z-index'
+        );
```

Comment:
Ping @jgroth: I'd like your opinion on whether you think this is ok to do on every render…

Maybe we should just do it when the component loads or something? You're probably not going to change it throughout the lifetime of the component, right? 🤔

Reply:
This is more safe in case it would change, but I agree that it feels very unlikely that you would change it once it has been set.

Also, I don't think that this would impact performance in a way that you would notice it. The component will not render several times per second, more likely only when you open/close it.

Reply:
Ok, that's great. Then let's go with this for now 👍